src/services/event.ts

```diff
@@ -173,8 +173,6 @@ async function findEventHelper(where, plural = false): Promise<EventShape | Even
       if (ownerUser) {
         owner = ownerUser.toJSON();
       }
-    } else {
-      owner = null;
```

Comment:
We can safely remove this branch because it shouldn't be possible to create an EventReportPilot *without* an ownerId, because omitting ownerId during creation causes a NotNullViolation.